scantools/scanners/navvis/iwconfig_parser.py

```diff
@@ -49,15 +52,17 @@ def parse_iwconfig(data):
                 'mac_address': r'Address:\s((?:[a-fA-F0-9]{2}[:|\-]?){6})',
                 'signal_level': r'Signal level=(-?\d{1,2}) dBm',
                 'frequency': r'Frequency:(\d{1,2}.\d{1,6}\s([ G|M|k]Hz))',
-                'time_offset': r'Extra: Last beacon: (\d{1,6})ms'
+                'time_offset': r'Extra: Last beacon: (\d{1,6})ms',
+                'ssid': r'ESSID:"(.{1,64})"'
```

Comment:
`ESSID` - sorry, not an expert - this contains same value as SSID below?

Reply:
yes. so ESSID and SSID are practically the same.